styles.css

```diff
@@ -136,14 +478,195 @@ ul.navbar-items {
+  }
+
+  /* works-section responsive style  */
+
+  #works-section {
+    margin-bottom: 100px;
+    overflow-x: hidden;
+  }
+
+  .box-container {
+    flex-direction: column;
+    margin: 5px 10px;
+  }
+
+  .left-section {
+    margin-bottom: 10px;
+  }
+
+  .left-section img {
+    float: none;
+    width: 100% !important;
+  }
+
+  .grid-container {
+    display: grid;
+    grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
+    gap: 10px;
+    margin: 0;
+    padding-left: 4%;
+    padding-right: 4%;
+  }
+
+  .first-desc {
+    text-align: left;
+    margin: 10px;
+  }
+
+  .first-desc h3 {
+    font-size: 16px;
+  }
+
+  .first-desc ul {
+    padding-left: 0;
+  }
+
+  .first-desc p {
+    font-size: 14px;
+  }
+
+  .first-desc ul li {
+    display: inline-block;
+    margin: 10px 0;
+    padding-left: 10px;
+  }
+
+  .first-desc button {
+    margin-top: 20px;
+    color: #f4f5f7;
+  }
+
+  .hr {
+    margin: 20px;
+    font-family: 'Crete Round', sans-serif;
+    color: #1c1a19;
+  }
+
+  hr {
+    position: relative;
+    top: 0;
+    left: 0;
+    width: 100%;
+    background-color: #212121;
+  }
+
+  .item-child button {
+    display: block;
+    background-color: #f55800;
+    width: 100%;
+    height: 50px;
+    margin-bottom: -30px;
+  }
+
+  .grid-btn {
+    border: none;
+    font-family: 'Crete Round', sans-serif;
+    font-size: 17px;
+    color: #f4f5f7;
+    text-align: center;
+  }
+
+  /* about-section responsive styles */
+
+  #about-page-container {
+    background-color: #1c1a19;
+    padding: 20px;
+  }
+
+  .about-me-container {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 10px;
+  }
+
+  .about-me-info {
+    margin: 50px 0 0 0;
+    padding: 0 10px;
+  }
+
+  .about-me-info h2 {
+    font-size: 30px;
+    margin-bottom: 10px;
+    color: #f55800;
+  }
+
+  .about-me-info p {
+    color: #f4f5f7;
+    font-size: 18px;
+  }
+
+  .about-me-info button {
+    color: white;
+    padding: 10px;
+    background-color: #f55800;
+    border: none;
+  }
+
+  .about-me-image {
+    flex: 1;
+    width: 100%;
+    height: auto;
+    margin-right: 250px !important;
+  }
+
+  .about-flex-container {
+    display: flex;
+    flex-direction: column;
+    gap: 10px;
+    margin-top: 30px;
+    margin-right: 0;
+  }
+
+  .about-flex-item {
+    background-color: #1c1a19;
+    padding: 20px;
+    text-align: center;
+  }
+
+  .about-flex-item h3 {
+    font-size: 24px;
+    margin-bottom: 10px;
+    position: relative;
+    padding-left: 25px;
+    color: white;
+  }
+
+  .about-flex-item:nth-child(1) h3::before {
+    background-color: #f55800;
+  }
+
+  .about-flex-item:nth-child(2) h3::before {
+    background-color: #f55800;
+  }
+
+  .about-flex-item:nth-child(3) h3::before {
+    background-color: #f55800;
+  }
+
+  .about-flex-item ul li {
+    margin-bottom: 15px;
+    list-style: none;
+    color: white;
```

Comment:
- **[OPTIONAL]** Please 🙏🏽 use the correct font family and styles to be similar to the Figma template 👇🏽

![image](https://github.com/Xghayor/Portfolio/assets/58642949/7479025f-b4a4-4ee1-a97f-6123123b3480)